code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -818,6 +823,10 @@
 			user << "<span class='warning'>You can only enthrall humans.</span>"
 			revert_cast()
 			return
+		if(iswizard(target))
+			user <<"<span class='warning'>Your tendrils discover truths in this man's mind that are forbidden even to you! Your mind recoils in horror at knowledge beyond your form before you can enslave [target].</span>"
```

Comment:
"The term man (from Proto-Germanic *mannaz or *manwaz "man, person") and words derived from it can designate any or even all of the human race regardless of their sex or age. "

It should probably still be changed however.